### DISCUSSION TITLE
New releases - who's managing them?

### DISCUSSION THREAD

**Discussion by martinjlowm** [aws/aws-lambda-rust-runtime#937] [Q&A] [answered]
With the recent layoffs, who's currently in charge of managing new releases? We're eagerly waiting a new release of the Lambda events crate - it's been 6 months since 0.15.1.

**Comment by bnusunny** [comment, marked as answer]
Oh, sorry. I will check that and push a new release soon.

**Reply by martinjlowm** [reply]
Thanks! Appreciate the swift reply :)

**Reply by martinjlowm** [reply]
Hate to be that guy @bnusunny - any update on this?

**Reply by bnusunny** [reply]
Hi, I plan to get the release out this weekend. Hope that's not too late.

**Reply by martinjlowm** [reply]
All good! Much appreciated - we would normally just have checked out the commit directly as a git dependency, but we use Nix (crate2nix) to manage all of our dependencies and that setup currently doesn't support Cargo workspaces.

**Reply by martinjlowm** [reply]
@bnusunny 👋

I noticed the pull request that was recently approved and merged - looks like the release is a manual task afterwards 🙏

**Reply by bnusunny** [reply]
@martinjlowm Sorry for the late update. I pushed the releases to crates.io. I was bogged down by re:Invent.

**Reply by martinjlowm** [reply]
No problem! Thanks a lot! I had a gut feeling that something like that could be going on. I hope you enjoyed the conference - so many people - I wonder if I saw you there 😃

**Reply by bnusunny** [reply]
We might!
